src/main/resources/templates/index.html

```diff
@@ -176,8 +186,10 @@ <h1 id="view">Login</h1>
 	<div id="user">User</div>
 	<div id="email">email</div>
 	<div id="password">password</div>
-	
-	<form id="form2" action="#" th:action="@{/registerCoach}" method="GET">
+</div>
+<br>

```

Comment:
As long as you are using CSS, avoid using br after a div.  You can accomplish the same thing by using CSS margin.